gprofiler/profilers/java.py

```diff
@@ -303,7 +308,7 @@ def _get_stop_cmd(self, with_output: bool) -> List[str]:
             ap_params.append(self.OUTPUT_FORMAT)
             ap_params.append(self.FORMAT_PARAMS)
         ap_params.append(f"log={self._log_path_process}")
-        return self._get_base_cmd() + [",".join(ap_params)]
+        return self._get_base_cmd() + [",".join(ap_params) + self._get_extra_ap_args()]
```

Comment:
hmmmm... we sure we want to pass those same arguments on stop? What is the use case here? Aren't most arguments we might pass only relevant to `start` cmd?

Reply:
Some arguments are relevant for `stop` (e.g the output file, output type, output format). In any case, AP doesn't care if you give arguments in start/stop; IIRC `profiler.sh` does that as well.

And actually, until [this](https://github.com/jvm-profiling-tools/async-profiler/pull/503) PR, AP wouldn't even complain on unsupported args.

Reply:
The idea of this change is to support everything we might want

Reply:
I see, so even now it just silently ignores args that are inappropriate for the current action :thinking: , oh well, if we ever need it we will just separate it to 2 different args (extra_start and extra_stop

Reply:
I doubt we will